invenio_rdm_records/records/jsonschemas/records/definitions-v2.0.0.json

```diff
@@ -406,7 +406,17 @@
     "additionalProperties": false,
     "properties": {
       "user": {
-        "type": "integer"
+        "type": ["string", "integer"]
```

Comment:
allowing `string` as well, in order to be able to represent `{"user": "system"}`